services/core-api/app/api/utils/helpers.py

```diff
@@ -1,6 +1,21 @@
 import re
 
+
 def clean_HTML_string(raw_html):
-  cleanr = re.compile('<.*?>')
-  cleantext = re.sub(cleanr, '', raw_html)
-  return cleantext
\ No newline at end of file
+    cleanr = re.compile('<.*?>')
+    cleantext = re.sub(cleanr, '', raw_html)
+    return cleantext
+
+
+"""
+Add sufix to permit number adding one to versioning.
+"""
+
+
+def generate_permit_no_sufix(permit, separator, filling=2):
```

Comment:
Typo, sufix --> suffix

Reply:
So, this is used just for determining what a draft permit's permit number should be? I would document that maybe, otherwise it could be used for issuing permits or cause other confusion.

Reply:
Absolutely Luke :)